docs/FAQs.md

```diff
@@ -1,12 +1,6 @@
 # FAQs
 
-## Contents
```

Comment:
I personally kind of like the contents sections, but if more people like it the other way I am happy to let myself be outvoted. Maybe an alternative is to keep the contents but bump the headling levels to be all `##` so that it shows up uniformly on the index page?